Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -943,7 +943,8 @@ def audit_revision_and_version_scheme
 
       if current_version == newest_committed_version &&
          current_url == newest_committed_url &&
-         current_checksum != newest_committed_checksum
+         current_checksum != newest_committed_checksum &&
+         !current_checksum.nil? && !newest_committed_checksum.nil?
```

Comment:
```suggestion
         current_checksum.present? && newest_committed_checksum.present?
```

In general `present?` and `blank?` are preferable to `!nil?` and `nil?` because they handle empty strings (and generally avoiding negation makes it easier to read).